Neos.Neos/Classes/Fusion/ContentElementWrappingImplementation.php

```diff
@@ -95,9 +95,18 @@ protected function getContentElementFusionPath()
             && isset($fusionPathSegments[$numberOfFusionPathSegments - 2])
             && $fusionPathSegments[$numberOfFusionPathSegments - 2] === 'process') {
 
-            // cut of the processing segments "__meta/process/contentElementWrapping<Neos.Neos:ContentElementWrapping>"
+            // cut off the SHORT processing syntax "__meta/process/contentElementWrapping<Neos.Neos:ContentElementWrapping>"
             return implode('/', array_slice($fusionPathSegments, 0, -3));
         }
+
+        if (isset($fusionPathSegments[$numberOfFusionPathSegments - 4])
+            && $fusionPathSegments[$numberOfFusionPathSegments - 4] === '__meta'
+            && isset($fusionPathSegments[$numberOfFusionPathSegments - 3])
+            && $fusionPathSegments[$numberOfFusionPathSegments - 3] === 'process') {
+
+            // cut off the LONG processing syntax "__meta/process/contentElementWrapping/expression<Neos.Neos:ContentElementWrapping>"
+            return implode('/', array_slice($fusionPathSegments, 0, -4));
+        }
```

Comment:
make sense, although could possibly be done in a more generic and simple way by removing everything starting from the `__meta` key